Coin toss steps update

gyan0890 opened the pull request:
I have updated the coin-toss-game README.md with the appropriate steps required to run this tutorial. I have also added the aenode configurations that need to be made before running this demo.

gyan0890 committed:
Added step-by-step instructions to run the tutorial including
settings for running the aenode.

gyan0890 committed:
Update README.md

omar-saadoun reviewed (commented):
When a comment like this appears : 
`Mac Users: For the first build, you might increase the RAM allocated to your docker in the docker desktop menu to at least 2,5 GB of free memory.`
Could you add or point a link where the command is showed?

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+a. Run an aeternity node
+- You can run a local aeternity node. You can choose to run a node on your local machine or you can use docker to run the node. You can also run it on another machine but you may have to change some settings in the ae-channel-service if you do so.
+- [Here](https://blog.aeternity.com/why-run-an-ae-node-and-how-to-do-it-8b95a685f683) is an article that describes how to run your own node.
+- While running the node, it is important to understand that your node should have an access to the WebSocket endpoint (by default it lives on 127.0.0.1:3014). If you run a docker image - you must provide a proper setup to expose it out of the docker. Note that by default it is bound to the localhost of the docker image.
```
```suggestion
- While running the node, it is important to understand that your node should have access to the WebSocket endpoint (by default it lives on 127.0.0.1:3014). If you run a docker image, you must provide a proper setup to expose it out of the docker. Note that by default it is bound to the localhost of the docker image.
```

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+- You can run a local aeternity node. You can choose to run a node on your local machine or you can use docker to run the node. You can also run it on another machine but you may have to change some settings in the ae-channel-service if you do so.
+- [Here](https://blog.aeternity.com/why-run-an-ae-node-and-how-to-do-it-8b95a685f683) is an article that describes how to run your own node.
+- While running the node, it is important to understand that your node should have an access to the WebSocket endpoint (by default it lives on 127.0.0.1:3014). If you run a docker image - you must provide a proper setup to expose it out of the docker. Note that by default it is bound to the localhost of the docker image.
+- In order to enforce the websocket endpoint opening at 3014, you can use either of the following .yaml file for your node configuration:
```
```suggestion
- In order to enforce the websocket endpoint opening at 3014, you can use either of the following .yaml files for your node configuration:
```

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+    -v ~/.aeternity/myaeternity.yaml:/home/aeternity/.aeternity/aeternity/aeternity.yaml \
+    aeternity/aeternity
+
+- Once your node is up and running, you would also need to create accounts. In real life that would be the result of the ICO(accounts) but for the test you can set your own.
```
```suggestion
- Once your node is up and running, you will also need to create accounts. In real life that would be the result of the ICO(accounts) but for the test you can set your own.
```

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+
+- Once your node is up and running, you would also need to create accounts. In real life that would be the result of the ICO(accounts) but for the test you can set your own.
+How can you set the accounts?
+Inside your node directory(the nodee that you are running), you will find a file called 'accounts_test.json' under the path - /node/data/aecore/.genesis
```
```suggestion
Inside your node directory (the node that you are running), you will find a file called `accounts_test.json` under the path `/node/data/aecore/.genesis`
```

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+- Once your node is up and running, you would also need to create accounts. In real life that would be the result of the ICO(accounts) but for the test you can set your own.
+How can you set the accounts?
+Inside your node directory(the nodee that you are running), you will find a file called 'accounts_test.json' under the path - /node/data/aecore/.genesis
+You can replace the accounts_test.json with [this](https://github.com/aeternity/ae-channel-service/blob/master/test/accounts_test.json) file:
```
```suggestion
You can replace the `accounts_test.json` with [this](https://github.com/aeternity/ae-channel-service/blob/master/test/accounts_test.json) file:
```

gyan0890 replied:
Updated all and committed the changes.

uwiger commented on README.md:
```diff
@@ -1,6 +1,34 @@
+
+The above json has public keys indicating the accounts as well as their balance in aettos.
+
+b. You need to install and the AE Channel Service. The AE Channel Service will serve as an automated counterpart that will play the Coin Toss Game against you. Follow [this](https://github.com/aeternity/ae-channel-service) link to install and run the ae-channel-service. Mac Users: For the first build, you might increase the RAM allocated to your docker in the docker desktop menu to at least 2,5 G ...
```
```suggestion
b. You need to install and run the AE Channel Service. The AE Channel Service will serve as an automated counterpart that will play the Coin Toss Game against you. Follow [this link](https://github.com/aeternity/ae-channel-service) to install and run the ae-channel-service. Mac Users: For the first build, you might increase the RAM allocated to your docker in the docker desktop menu to at least 2,5 GB of free memory.
```

uwiger commented on README.md:
```diff
@@ -19,6 +47,13 @@ The application automatically points to your Aeternity Node and AE Channel servi
 
 In case of questions, issues and featur requests, please turn to https://forum.aeternity.com/ .
 
+3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. You must configure your ae-channel-service properly so it can sign your transactions for you. Sadly, the demo client does not do that so you must fill your key pair here. If you’re running a local standalone test, you must configure your accounts_test.json accordingly.
```
This is the same as in 2 c above.

gyan0890 committed:
Update README.md

Made some changes as suggested by @uwiger and @omar.

gyan0890 committed:
Add files via upload

Added a screenshot of the docker properties for Macbook users running the node using docker.

gyan0890 committed:
Update README.md

Added the link to the docker properties image.

omar-saadoun commented:
Hi @uwiger can you check your comments and approve this PR ? Thanks!

uwiger commented on README.md:
```diff
@@ -47,12 +47,12 @@ The application automatically points to your Aeternity Node and AE Channel servi
 In case of questions, issues and featur requests, please turn to https://forum.aeternity.com/ .
 
-3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. You must configure your ae-channel-service properly so it can sign your transactions for you. Sadly, the demo client does not do that so you must fill your key pair here. If you’re running a local standalone test, you must configure your accounts_test.json accordingly.
+3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. For each of the environmeents, you must configure your ae-channel-service properly so it can sign your transactions for you.
```
```suggestion
3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. For each of the environments, you must configure your ae-channel-service properly so it can sign your transactions for you.
```

uwiger commented on README.md:
```diff
@@ -47,12 +47,12 @@ The application automatically points to your Aeternity Node and AE Channel servi
 
 5. For the node config files:
-An example config files can be found in the ae-channel-service repo. There are two different setups, the difference between them is simply if the node mines with a normal speed or much faster. You can find the fast config here and the normal one here. Please note that if you want iris enabled, you shall add a corresponding line in the hard_forks section.
+An example config files can be found in the ae-channel-service repo. There are two different setups, the difference between them is simply if the node mines with a normal speed or much faster(the config files are linked in 1a). Please note that if you want iris enabled, you shall add a corresponding line in the hard_forks section.
```
```suggestion
Examples of config files can be found in the ae-channel-service repo. There are two different setups: the difference between them is simply if the node mines with a normal speed or much faster (the config files are linked in 1a). Please note that if you want iris enabled, you shall add a corresponding line in the `hard_forks` section.
```

omar-saadoun commented:
@gyan0890 please check on Ulf's commnets

gyan0890 committed:
Update README.md

Co-authored-by: Ulf Wiger <ulf@wiger.net>

gyan0890 committed:
Update README.md

Co-authored-by: Ulf Wiger <ulf@wiger.net>

gyan0890 commented:
> @gyan0890 please check on Ulf's comments

Done, updated the commit :)

omar-saadoun commented:
thanks @gyan0890 , @uwiger can you approve it? Thanks!

omar-saadoun commented:
@uwiger can you merge this PR ? Or @nikita-fuchs ?
